### PR TITLE
feat: Add LoRA-based knowledge distillation for Wan-Video

### PR DESCRIPTION
**Pull request by MaulanaYusufIkhsanRobbani**
This commit introduces a new training script and a shell script to perform LoRA-based knowledge distillation for Wan-Video models.

The new script `examples/wanvideo/model_training/train_distill.py` is a modified version of the standard training script. It is designed to load both a student and a teacher model. The student model is trained with a LoRA adapter, and the loss function is a combination of the standard diffusion loss and a distillation loss. The distillation loss is the mean squared error between the student's and the teacher's predictions.

A new shell script `examples/wanvideo/model_training/lora/Wan-Distill.sh` is provided to launch the distillation process with the correct arguments.

### CODE REVIEW

**Comment by Artiprocher** [comment]
@MaulanaYusufIkhsanRobbani Thank you for providing the code. Recently, we have been researching how to build a more general training framework that encapsulates SFT, distillation, and DPO into a unified module, making it orthogonal to the model architecture. Therefore, we will merge this functionality at a later time.
